### PR TITLE
[14.0][IMP] mrp_subcontracting_purchase_link: add post init hook to populate the PO line on previous manufacturing orders

### PR DESCRIPTION
**Pull request by JordiBForgeFlow**
Otherwise when you install the module the previous subcontracting MO's will not have the link with the purchase order
cc @ThiagoMForgeFlow

### CODE REVIEW

File: mrp_subcontracting_purchase_link/hooks.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Copyright 2023 ForgeFlow, S.L.
+# License OPL-1
```
Comment (yostashiro) [inline mrp_subcontracting_purchase_link/hooks.py]: Should be LGPL, I suppose.

Comment (JordiBForgeFlow) [reply mrp_subcontracting_purchase_link/hooks.py]: @yostashiro You are right! Fixed ;)

**Review by yostashiro — APPROVED** [review]
Code review. LGTM. 👍

**Review by LoisRForgeFlow — APPROVED** [review]

**Comment by LoisRForgeFlow** [comment]
Good morning ocabot, time to get up! :)

/ocabot merge minor

**Pull request merged**